### PR TITLE
Improvements to fonts and buttons

### PR DESCRIPTION
**Pull request by magska**
The button changes are a bit risky. I have been through all of datahub to see that every button looks ok. But it could be that there are still some types of buttons that I havent yet encountered. No risk to functionality. So those can be fixed quickly as they appear.

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by adriaanslechten — APPROVED** [review]

**Comment by adriaanslechten** [comment]
I'm thinking if we are going to add more overwrites we should maybe keep an overview in the docs or so. (mainly about the why each part is overwritten, as a simple diff just shows what is overwritten.)

**Comment by magska** [comment]
@adriaanslechten agreed. Do we have a good place to document?

**Comment by adriaanslechten** [comment]
Maybe just in the README of this project? We got a confluence page as well but it might be nicer to keep it near the code?
@thomasplarsson / @RickardCardell  ?

**Comment by magska** [comment]
Maybe I can have an overrides.md in the react folder or something.

**Comment by thomasplarsson** [comment]
An overrides.md sounds good to me.

**Review by thomasplarsson — APPROVED** [review]
LGTM.

File: datahub-web-react/public/index.html [hunk 1]
```diff
@@ -20,7 +22,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-        <title>DataHub</title>
+        <title>Lakehouse Catalog</title>
```
Comment (thomasplarsson) [inline datahub-web-react/public/index.html:25]: Should perhaps be Lakehouse Data Catalog.

Comment (magska) [reply datahub-web-react/public/index.html:25]: Cool. Will fix

**Pull request merged**